### DISCUSSION TITLE
유니티에서 셋팅한 라이트와 실제로 적용된 라이트 빛이 다릅니다

### DISCUSSION THREAD

**Discussion by icjoon** [naverz/zepeto-studio-kor#522] [5. Q&A]
![라이트문제-1](https://user-images.githubusercontent.com/86340067/159640777-2ddc5ba9-5c1d-47b6-93f5-c9d0d58a1f6a.PNG)
![라이트문제-2](https://user-images.githubusercontent.com/86340067/159640787-4965fb9b-2d10-4e25-a1d9-0bb1d9859e08.jpg)

적용이 안되는 건 아닌 것 같은데 라이트 셋팅할 때 유의사항 같은게 있나요?

**Comment by x4cf** [comment]
제페토에서 재공하는 라이팅 설정을 사용하시기를 추천하며, 프로젝트 셋팅에서 모바일 화면과 같게 보이도록 설정하고 작업하기를 추천합니다.

유니티는 빌드타겟에 따라 라이팅 셋팅이 다르며 에디터 버전에서는 고퀄리티 환경이 기본으로 적용되어있습니다.
모바일 개발환경에 맞게 퀄리티 셋팅을 하고 개발하면 개발시 화면과 모바일 출력화면의 싱크로를 맞추기 편합니다.

1. 제페토 기본 라이팅 (주의 : 누르는것과 동시에 적용되고 이전으로 되돌릴 수 없음, 기존 라이트 셋팅에 문제가 될 수 있으니 백업 후 진행 추천)
<img width="605" alt="라이팅 설정" src="https://user-images.githubusercontent.com/90679926/159647815-9a8fce0c-917e-4ad6-b152-34883dd53ab6.png">

2. 프로젝트 셋팅 => 퀄리티 ( 이곳에서 모바일 퀄리티 셋팅을 진행해주세요 :D )
<img width="943" alt="프로젝트 셋팅" src="https://user-images.githubusercontent.com/90679926/159648081-68177670-4042-4770-a810-93582e293e0b.png">

**Reply by icjoon** [reply]
사진과 똑같이 맞췄더니 셋팅한 라이팅이 그대로 적용 되는걸 확인했습니다 감사합니다!
